offload/DeviceRTL/src/Debug.cpp

```diff
@@ -26,10 +26,13 @@ using namespace ompx;
 extern "C" {
 void __assert_assume(bool condition) { __builtin_assume(condition); }
 
+#ifndef OMPTARGET_HAS_LIBC
```

Comment:
No `OMPTARGET` anymore I suggest.

Reply:
I think there's a lot of other stuff we'd need to clean up so it'd be easier to do that all at once.

Reply:
Even though we have a lot but they are "legacy". I don't think it's a good idea to introduce more.

Reply:
This is an existing macro, so if I wanted to use something else I'd need to rename it which would make the patch more complicated.

Reply:
Oh didn't notice that. It is indeed an existing one. Okay. Renaming belongs to a separate patch.

Reply:
Yeah we should probably do it all at once.